assignment/scripts/describe.js

```diff
@@ -91,6 +93,14 @@ if (isStudent === true && zip > 80000 ) {
   console.log('How about the weather?')
 }
 
+//Setting a variable isStudent equal to true and the variable age equal to 34 and the variable zip equal to 55407
+//if isStudent is strictly equal to 2 and zip is greater than 80000, console.log('you are a student on the West Coast!')
+//else if isStudent is strictly equal false or age < 30, console.log('What are your hobbies?')
```

Comment:
This is starting to look more like JavaScript than english. The assignment is to take the JavaScript and translate it into plain language.